src/replicated.ts

```diff
@@ -248,11 +248,21 @@ export interface ContentTrust {
 // backup
 /*
 type Backup struct {
-	Enabled         string `yaml:"enabled" json:"enabled"`
-	Hidden          string `yaml:"hidden" json:"hidden"`
-	PauseAll        bool   `yaml:"pause_all" json:"pause_all"` // deprecated
-	PauseContainers string `yaml:"pause_containers" json:"pause_containers"`
-	Script          string `yaml:"script" json:"script"`
+    Enabled         string       `yaml:"enabled" json:"enabled"`
```

Comment:
is this just dead code?

Reply:
Yeah this whole file can probably be dumped. We we're gonna try to generate schema from typescript but I don't think it's worth it. We currently generate from golang